### PR TITLE
add auto-confirm option for `create` subcommand

### PR DESCRIPTION
**Pull request by igalic**
this pr addresses #202

### CODE REVIEW

File: iocage/cli/create.py [hunk 1]
```diff
@@ -98,13 +100,14 @@ def cli(release, template, count, props, pkglist, basejail, empty, short,
...
             })
         except ValueError:
-            ioc_common.logit({
-                "level"  : "WARNING",
-                "message": "This may be a short UUID, template creation only"
-                           " supports TAGs"
-            })
-            if not click.confirm("\nProceed?"):
-                exit()
+            if not force:
```
Comment (gronke) [inline iocage/cli/create.py:103]: force is not defined yet. Did you forget to put in in the cli method params?

File: iocage/cli/create.py [hunk 2]
```diff
@@ -98,13 +100,14 @@ def cli(release, template, count, props, pkglist, basejail, empty, short,
...
+                ioc_common.logit({
+                    "level"  : "WARNING",
+                    "message": "This may be a short UUID, template creation only"
```
Comment (gronke) [inline iocage/cli/create.py]: The linter requires lines to be max 80 characters long. I suggest to shift a bit more of the text to the next line.

File: iocage/cli/create.py [hunk 3]
```diff
@@ -98,13 +100,14 @@ def cli(release, template, count, props, pkglist, basejail, empty, short,
...
+                            " supports TAGs"
```
Comment (gronke) [inline iocage/cli/create.py]: From Tavis CI:
```
+./iocage/cli/create.py:107:29: E131 continuation line unaligned for hanging indent
```

Seems this line needs to be indented on a different level.

**Review by gronke — COMMENTED** [review]
👍 besides the few things the linter complains about :)

File: iocage/cli/create.py (hunk 2 above)
Comment (igalic) [reply iocage/cli/create.py]: my personal preference is to have error messages in-tact, to make grepping for them easier
failing that, i think the next best would be to break on `,` in this case.

File: iocage/cli/create.py (hunk 1 above)
Comment (igalic) [reply iocage/cli/create.py:103]: yeeeaah… i did… i was looking for where this is passed, and decided, it's done by magic.
thanks for the clarification ;)

File: iocage/cli/create.py (hunk 2 above)
Comment (skarekrow) [reply iocage/cli/create.py]: Breaking on `,` in this case is fine

**Comment by skarekrow** [comment]
Once you fix the things @gronke (<3) reviewed, I'll merge it in :)

**Comment by igalic** [comment]
i'm still struggling with my dev setup here… last bit seems to be [yapf's lack for 3.6 support](https://github.com/google/yapf#python-versions)

**Review by skarekrow — APPROVED** [review]
Changing the `:` in this isn't the issue ;) My editor will auto move these back as I like them to be symmetrical.

The flake issue is that there are multiple spaces *after* not before the `:`, but this is something I can easily fix so no worries.

**Comment by skarekrow** [comment]
Thanks again @igalic 👍

**Pull request merged**